pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsXml.scala

```diff
@@ -173,18 +192,27 @@ case class MetsXml(root: Elem) {
     *
     *  For this input we would expect the following output:
     *
-    *  Map("FILE_0001_OBJECTS" -> "objects/b30246039_0001.jp2",
-    *      "FILE_0002_OBJECTS" -> "objects/b30246039_0002.jp2")
+    *  Map("FILE_0001_OBJECTS" -> FileObject("FILE_0001_OBJECTS", "objects/b30246039_0001.jp2", "image/jp2"),
+    *      "FILE_0002_OBJECTS" -> FileObject("FILE_0002_OBJECTS", "objects/b30246039_0002.jp2", "image/jp2"))
     */
-  private def fileObjects: Map[String, String] =
+  private def fileObjects: ListMap[String, FileObject] =
     (root \ "fileSec" \ "fileGrp")
       .filterByAttribute("USE", "OBJECTS")
       .childrenWithTag("file")
-      .toMapping(
-        keyAttrib = "ID",
-        valueNode = "FLocat",
-        valueAttrib = "{http://www.w3.org/1999/xlink}href"
-      )
+      .map { file =>
```

Comment:
I think you may need to rebase from master. I've done something very similar to this in https://github.com/wellcometrust/catalogue/pull/398

Reply:
Ha, at least we're thinking along the same lines. Will do.